server_src/routes/settings.js

```diff
@@ -12,24 +12,13 @@ const UserDB = require("../models/User.js");
 const PrinterDB = require("../models/Printer.js");
 const AlertsDB = require("../models/Alerts.js");
 const GcodeDB = require("../models/CustomGcode.js");
-
 const Logger = require("../lib/logger.js");
-
 const logger = new Logger("OctoFarm-API");
-
 const runner = require("../runners/state.js");
 const multer = require("multer");
-
 const { Runner } = runner;
-
-const systemInfo = require("../runners/systemInfo.js");
-
-const SystemInfo = systemInfo.SystemRunner;
-
-const settingsClean = require("../lib/dataFunctions/settingsClean.js");
-
-const { SettingsClean } = settingsClean;
-
+const { SystemRunner } = require("../runners/systemInfo.js");
```

Comment:
I tend to simplify the includes and make them as compact as possible.
Also from other languages Ive learned to put node_modules at the top, followed by our own modules.